src/main/java/org/folio/innreach/client/InventoryClient.java

```diff
@@ -22,4 +24,10 @@
 
   @GetMapping("/items?query=hrid=={hrId}")
   ResultList<InventoryItemDTO> getItemsByHrId(@PathVariable("hrId") String hrId);
+
+  @GetMapping("/items?query=barcode=={barcode}")
+  InventoryItemDTO getItemByBarcode(@PathVariable("barcode") String barcode);
+
+  @PutMapping("/items/{itemId}")
+  void updateItem(@PathVariable("itemId") UUID itemId, @RequestBody InventoryItemDTO inventoryItem);
```

Comment:
I would suggest to use inventory-storage API instead. Inventory and inventory storage records have a bit different structure and I think it's better to keep using one API as long as it is reasonable. Right now we use inventory storage API for initial record contribution and for some of other logic and it looks much more suitable from backend perspective as it only loads identifiers of reference data without their details.

Reply:
@oliinyko thanks, I'll check